src/main/java/frc/robot/subsystems/PneumaticsSubsystem.java

```diff
@@ -9,9 +9,9 @@
 import frc.robot.RobotLifecycleCallbacks;
 
 public class PneumaticsSubsystem extends SubsystemBase implements RobotLifecycleCallbacks {
-    DoubleSolenoid solenoid1 = new DoubleSolenoid(Ids.TEST_PCM_ID, PneumaticsModuleType.CTREPCM, Ids.TEST_SOLENOID1_FORWARD,
+    DoubleSolenoid solenoid1 = new DoubleSolenoid(Ids.PCM_ID, PneumaticsModuleType.CTREPCM, Ids.TEST_SOLENOID1_FORWARD,
```

Comment:
Maybe add `private final` to these, but it wasn't there before so nbd if we just want to leave it.